clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/JDBCStoreResourceDefinition.java

```diff
@@ -93,11 +93,11 @@
         private final AttributeDefinition definition;
 
         Attribute(String name, ModelType type, CapabilityReferenceRecorder reference) {
-            this.definition = createBuilder(name, type, true).setAllowExpression(false).setCapabilityReference(reference).build();
+            this.definition = createBuilder(name, type, true).setAllowExpression(false).setCapabilityReference(reference).setAlternatives(DeprecatedAttribute.DATASOURCE.getName()).build();
```

Comment:
That means that DeprecatedAttribute.DATASOURCE is also an alternative to DIALECT. Is that correct?